src/librssguard/miscellaneous/pluginfactory.cpp

```diff
@@ -77,7 +77,12 @@ QStringList PluginFactory::pluginPaths() const {
   paths << QCoreApplication::applicationDirPath() + QDir::separator() + QL1S("..") + QDir::separator() +
              QL1S(RSSGUARD_LIBDIR) + QDir::separator() + QL1S(APP_LOW_NAME);
 #elif defined(Q_OS_WIN)
+#ifdef _MSC_VER
   paths << QCoreApplication::applicationDirPath() + QDir::separator() + QL1S("plugins");
+#else
+  paths << QCoreApplication::applicationDirPath() + QDir::separator() + QL1S("..") + QDir::separator() +

```

Comment:
Well, let me step in here.

The location of RSS Guard runtime plugins must be the same on the particular OS regardless of which toolkit was used to compile the application.

While it makes sense to have plugins in Linux-like folder hierarchy on Linux, I do not want it on Windows.

Reply:
Ah.. that is unfortunate :( because the MSYS2 team will surely patch it back to FHS, downstream

Reply:
Wait, could you come to my discord channel to discuss this?

Reply:
https://discord.gg/XBxeDxzu

Reply:
Ok.. will ping you afterwards.. :)